index.html

```diff
@@ -39,5 +40,144 @@ <h2>LET'S CONNECT</h2>
         </ul>
         
     </section>
-</body>
-</html>
\ No newline at end of file
+    <section class="project-container">
```

Comment:
Please you've learned the Grid CSS so it would be better if you use Grid  CSS in the projects container

Reply:
thank you i resolved this too